Mathlib/RingTheory/Ideal/Cotangent.lean

```diff
@@ -109,7 +109,7 @@ theorem to_quotient_square_comp_toCotangent :
   LinearMap.ext fun _ => rfl
 #align ideal.to_quotient_square_comp_to_cotangent Ideal.to_quotient_square_comp_toCotangent
 
--- @[simp] -- Porting note: not in simpNF
+@[simp]
```

Comment:
I presume you checked that this differs from the lemma that would be generated by simps.
(If not, I'd suggest replacing this by the auto-generated one.)

Reply:
This is an ordinary simp lemma and simps cannot generate this.

Reply:
:bulb: Makes sense!